packages/modules/desktop-viewer-react/src/components/Viewer.tsx

```diff
@@ -10,6 +10,6 @@ import { useDesktopViewerInitializer } from "../hooks";
 import type { DesktopViewerProps } from "../types";
 
 export const Viewer = (props: DesktopViewerProps) => {
-  const initialized = useDesktopViewerInitializer(props);
+  const initialized = useDesktopViewerInitializer({...props, clientId: ""});
```

Comment:
Won't this override the `clientId` to always be blank? Desktop apps still need to provide a `clientId` if they're not using a local snapshot/briefcase.